### PR TITLE
CE & AI Awards

### PR DESCRIPTION
**Pull request by cheekybrdy**
Remade previous PR due to merge conflict problems

## About the PR 
Adds framed awards for the CE & AI. The first is to give a reason for antags to break into CE office & so there are reasons to breach into upload/core seperated maps without murderous intent

## Why's this needed? 
^ Fills in the gaps to the framed awards for every head

## Changelog

```changelog
(u)Cheekybrdy
(*)Added framed awards for the CE & AI
```

### CODE REVIEW

File: code/obj/decal/poster.dm [hunk 1]
```diff
@@ -1046,6 +1046,41 @@
...
+			icon_glass = "aiscript1"
+			icon_award = "aiscript"
+			icon_empty = "frame"
+			icon_state = "aiscript"
+
+			get_award_text(var/datum/mind/M)
+				var/ainame = "Anonymous"
+				if(M?.current?.client?.preferences?.name_last)
+					ainame = M.current.client.preferences.name_last
+				. += "It says \ [ainame] has written and ran a working space-python script."
```
Comment (TobleroneSwordfish) [inline code/obj/decal/poster.dm:1082]:
```suggestion
				. += "It says \ [ainame] has written and run a working space-python script."
```

**Review by TobleroneSwordfish — CHANGES_REQUESTED** [review]
The changelog still says "CodeDude"

File: code/obj/decal/poster.dm (hunk 1 above)
Comment (cheekybrdy) [reply code/obj/decal/poster.dm:1082]: ran for past tense?

File: code/datums/gamemodes/spy_theft.dm [hunk 2]
```diff
@@ -334,8 +334,6 @@
 
 	station_bounties[/obj/item/disk/data/floppy/read_only/communications] = 2
 	station_bounties[/obj/item/disk/data/floppy/read_only/authentication] = 3
-	station_bounties[/obj/item/disk/data/floppy/manudrive/ai] = 2
```
Comment (cheekybrdy) [inline code/datums/gamemodes/spy_theft.dm:337]: mb I'll revert this

Comment (ZeWaka) [reply code/datums/gamemodes/spy_theft.dm:337]: let me know if you need any help in imcoder

File: code/obj/decal/poster.dm (hunk 1 above)
Comment (cheekybrdy) [reply code/obj/decal/poster.dm:1082]: should be good to go!

**Comment by cheekybrdy** [comment]
Should be good to go!

File: code/obj/decal/poster.dm (hunk 1 above)
Comment (TobleroneSwordfish) [inline code/obj/decal/poster.dm:1082]:
```suggestion
				. += "It says \ [ainame] has written and run a working space-python script."
```
https://ell.stackexchange.com/questions/69016/something-has-run-ran-its-course

**Comment by cheekybrdy** [comment]
mb toblerone

**Comment by flrsh** [comment]
Personally I'd like to see more unique things for heads / the AI rather than very similar awards (e.g. AI lawrack blueprint for the CE).

**Comment by cheekybrdy** [comment]
!merge_upstream

**Pull request closed without merging**